cluster/ci/monitoring/build_cop.md

```diff
@@ -0,0 +1,14 @@
+# build-cop dashboard in prow-monitoring
+
+The [build-cop dashboard](https://grafana-prow-monitoring.svc.ci.openshift.org/d/6829209d59479d48073d09725ce807fa/build-cop-dashboard?orgId=1) in [prow-monitoring](README.md) is an alternative tool which shows the sucessful rate for varous types of prow jobs in build-cop reports. The data in metrics are obtained by scaping the related component in prow. Every authenticated user of our CI cluster has the access to dashboard.
+
+[Prow deck](https://prow.svc.ci.openshift.org/) has been used for the similar purpose. Eg, deck page shows `Success rate over time: 3h: 78%, 12h: 81%, 48h: 77%` for job with name `*-master-e2e-aws`.
```

Comment:
Let's present this as a comparison: 

> The Build Cop must keep track of passing rates for a number of job types. Normally, this would be done by viewing a filtered list of job in `deck`. With the Grafana dashboards, though, an overview of all job types can be seen with one panel.

And give more detail and links as you see fit